chapters/methodology/theory.tex

```diff
@@ -234,3 +265,4 @@ \subsubsection{FW-CADIS-$\Omega$}
 and it may place too great of
 importance on the forward-moving particles in generating the variance reduction
 parameters.
+% May need to update this paragraph after we see what happens in practice.
```

Comment:
Agreed. I added this after a discussion I had with Douglas and wanted to make sure I acknowledged it at least somewhere.